Source/WebKit/UIProcess/API/Cocoa/WKWebViewTesting.mm

```diff
@@ -1076,6 +1076,15 @@ - (unsigned)_forwardedLogsCountForTesting
 #endif
 }
 
+- (unsigned)_launchLogsCountForTesting
+{
+#if ENABLE(LOGD_BLOCKING_IN_WEBCONTENT)
+    return _page->protectedLegacyMainFrameProcess()->logCountForTesting();
```

Comment:
Can you avoid calling `protectedLegacyMainFrameProcess()` in new code? We should probably be getting the process from the main frame instead.